static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,101 @@
+/* global SETTINGS: false */
```

Comment:
it'd be good to add flow to this as well, looks like it also passes without modification.